### PR TITLE
fix: community call

### PR DESCRIPTION
**Pull request by degenaro**
## Types of changes

- [ ] Hot fix (emergency fix and release)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation (change which affects the documentation site)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Release (`develop` -> `main`)

## Quality assurance (all should be covered).

- [ ] My code follows the code style of this project.
- [x] Documentation for my change is up to date?
- [ ] My PR meets testing requirements.
- [ ] All new and existing tests passed.
- [x] All commits are signed-off.

## Key links:

- [Sonar coverage](https://sonarcloud.io/dashboard?id=compliance-trestle)

# Before you merge

- Ensure it is a 'squash commit' if not a release.
- Ensure CI is currently passing
- Check sonar. If you are working for a fork a maintainer will reach out, if required.

### CODE REVIEW

**Comment by degenaro** [comment]
Reviewers - please suggest improvements, if any - thx!

File: README.md
Comment (jpower432) [inline README.md:123]: This should not block this particular change, but since the Slack and Zoom meeting are for the `oscal-compass` organization, I wonder if this type of information would belong in an organization profile or in a community repository.

File: README.md
Comment (jpower432) [inline README.md:121]: Perhaps we should add a process on how to get topics on the agenda

**Review by jpower432 — APPROVED** [review]
I left a couple of suggestions, but nothing that would block this change. Just discussion points.

**Comment by degenaro** [comment]
Dear Reviewers: please see changes since initial delivery. 
- every other Tue
- calendar link & instructions
- login notes
Comments?

File: README.md
Comment (degenaro) [reply README.md:130]: Where would this go, when I visit https://github.com/oscal-compass? Also, replicating information seemed doomed to inconsistency, so perhaps one should point to the other.

Comment (jpower432) [reply README.md:130]: We can have an organizational profile [READMEs](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile) so the information would be available when the organization link is visited. Example here for the CNCF org - https://github.com/cncf/.github/tree/master/profile. Totally agree that we should not duplicate the information, but link to it.

Comment (degenaro) [reply README.md:130]: So I think you mean here https://github.com/oscal-compass/.github in out case. Seems fine to me, but then perhaps we should open an issue against that repo and make a proposal as to what it should contain, pre-CNCF.

File: README.md
Comment (degenaro) [reply README.md]: I don't think we have an official process yet...but once we do we can certainly add words to the README.md community meetings section, if that make sense.

File: README.md
Comment (jpower432) [reply README.md:130]: Added here https://github.com/oscal-compass/.github/issues/1

**Comment by AleJo2995** [comment]
@degenaro Ash from CNCF has changed the zoom settings so we are able to use a Passcode instead of doing a login. I have added the Passcode in the latest commit as well as changed the wording for Zoom login, please take a quick look before merging.

**Review by AleJo2995 — APPROVED** [review]
LGTM

**Review by mrgadgil — APPROVED** [review]

**Pull request merged**
